.github/workflows/ci-model-regression-on-schedule.yml

```diff
@@ -0,0 +1,225 @@
+    - name: Checkout master
+      uses: actions/checkout@v2
+
+    - name: Download yq
+      run: |
+        curl --location https://github.com/mikefarah/yq/releases/download/3.3.0/yq_linux_amd64 -o yq
+        chmod +x yq
+
+    - run: test -f .github/configs/mr-test-schedule.yaml
+
+    - id: set-matrix
+      shell: bash
+      run: echo "::set-output name=matrix::$(./yq -j r .github/configs/mr-test-schedule.yaml)"
+
+  deploy_runner_gpu:
+    name: Deploy Github Runner - GPU
+    needs: read_test_configuration
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Download gomplate
+        run: |-
+          curl -o gomplate -sSL https://github.com/hairyhenderson/gomplate/releases/download/v3.6.0/gomplate_linux-amd64
+          chmod 755 gomplate
+
+      - name: Render deployment template
+        run: |-
+          ./gomplate -f .github/runner/deployment.yaml.tmpl -o runner_deployment.yaml
+
+      # Setup gcloud CLI
+      - uses: GoogleCloudPlatform/github-actions@0.1.2
+        with:
+          service_account_key: ${{ secrets.GKE_SA_RASA_CI_GPU }}
+          service_account_email: ${{ secrets.GKE_RASA_CI_GPU_SA_NAME }}
+
+      # Get the GKE credentials so we can deploy to the cluster
+      - run: |-
+          gcloud container clusters get-credentials "${{ secrets.GKE_GPU_CLUSTER }}" --zone "$GKE_ZONE" --project "${{ secrets.GKE_SA_RASA_CI_GPU_PROJECT }}"
+
+      - name: Deploy Github Runner
+        run: |-
+          kubectl apply -f runner_deployment.yaml
+          kubectl -n github-runner rollout status --timeout=15m deployment/github-runner-$GITHUB_RUN_ID
+
+  model_regression_test_gpu:
+    name: Model Regression Tests - GPU
+    continue-on-error: true
+    needs:
+    - deploy_runner_gpu
+    - read_test_configuration
+    env:
+      LD_LIBRARY_PATH: "/usr/local/cuda/extras/CUPTI/lib64:/usr/local/cuda/lib64:/usr/local/nvidia/lib:/usr/local/nvidia/lib64"
+      ACCELERATOR_TYPE: "GPU"
+    runs-on: [self-hosted, gpu]
+    strategy:
+      max-parallel: 1
+      matrix: ${{fromJson(needs.read_test_configuration.outputs.matrix)}}
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Checkout dataset
+        uses: actions/checkout@v2
+        with:
+          repository: ${{ secrets.DATASET_REPOSITORY }}
+          token: ${{ secrets.ML_TEST_SA_PAT }}
+          path: 'dataset'
+
+      - name: Set DATASET and CONFIG variables
+        env:
+          DATASET_NAME: "${{ matrix.dataset }}"
+          CONFIG_NAME: "${{ matrix.config }}"
+        run: |-
+          # determine DATASET and CONFIG environment variables
+          source <(gomplate -d mapping=./dataset/dataset_config_mapping.json -f .github/templates/configuration_variables.tmpl)
+          test -f dataset/configs/$CONFIG
+          test -d dataset/$DATASET
+          echo "::set-env name=DATASET::${DATASET}"
+          echo "::set-env name=CONFIG::${CONFIG}"
```

Comment:
do we really need the template for that?

`dataset_config_mapping.json` is included in the runner?

Reply:
Alas, we need to do something like this, `dataset_config_mapping.json` is a part of a different repository.